### PR TITLE
Vsharma tavisca flights

### PR DESCRIPTION
**Pull request by tavisca-vsharma**
Pull the code of my branch vsharma-tavisca-flights and review the code also

### CODE REVIEW

File: src/app/modules/batch/models/batchlistt.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export interface batchlistt {
```
Comment (lalit-tavisca) [inline src/app/modules/batch/models/batchlistt.ts]: follow pascal case

File: src/app/modules/batch/data/batchlistdata.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { batchlistt } from "./../models/batchlistt";
+import { DateAdapter } from "@angular/material/core";
```
Comment (lalit-tavisca) [inline src/app/modules/batch/data/batchlistdata.ts]: remove unused imports

File: src/app/modules/batch/components/batch-list/batch-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    // );
+  }
+  id: Number;
+  ngOnInit() {
+    this.batchlist.getBatchData(this.id).subscribe(
+      res => {
+        this.batches = res;
+      },
+      err => {
+        console.log(err);
```
Comment (lalit-tavisca) [inline src/app/modules/batch/components/batch-list/batch-list.component.ts]: setr this.batches = [];

File: src/app/modules/batch/components/batch-list/batch-list.component.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<body>
+  <div class="mat-app-background basic-container">
+    <div class="page-header">
+      <div class="right">
+        <h1 class="Team-Name-Header" matTooltip="Team Name">Team: {{team}}</h1>
```
Comment (lalit-tavisca) [inline src/app/modules/batch/components/batch-list/batch-list.component.html]: follow kebab case for css classes

File: src/app/modules/batch/models/batchlistt.ts (hunk 1 above)
Comment (tavisca-vsharma) [reply src/app/modules/batch/models/batchlistt.ts]: Done

File: src/app/modules/batch/components/batch-list/batch-list.component.ts (hunk 3 above)
Comment (tavisca-vsharma) [reply src/app/modules/batch/components/batch-list/batch-list.component.ts]: done

File: src/app/modules/batch/components/batch-list/batch-list.component.html (hunk 4 above)
Comment (tavisca-vsharma) [reply src/app/modules/batch/components/batch-list/batch-list.component.html]: done

**Pull request closed without merging**